cmd/utils/flags.go

```diff
@@ -513,6 +520,36 @@ func MakeAccountManager(ctx *cli.Context) *accounts.Manager {
 	return accounts.NewManager(ks)
 }
 
+// MakeExtra resolves extradata for the miner from a flag or returns a default
+func MakeExtra(ctx *cli.Context) []byte {
+	if ctx.IsSet(ExtraDataFlag.Name) {
+		return []byte(ctx.GlobalString(ExtraDataFlag.Name))
+	} else {
+		return makeDefaultExtra()
```

Comment:
You don't need the else here. The `if` branch already returns, so it's just unneeded boilerplate to have an else branch. Simply `return`.